template/{% if has_backend %}backend{% endif %}/tests/unit/test_fast_api_exception_handlers.py

```
@@ -0,0 +1,149 @@
+        assert response_json["status"] == codes.NOT_FOUND
+        assert response_json["detail"] == "Not Found"
+        assert response_json["errorType"] == "HTTPException"
+        assert response_json["instance"] == f"urn:uuid:{expected_uuid}"
+        self.spied_logger_warning.assert_called_once()
+        log_call_args = self.spied_logger_warning.call_args[0]
+        log_message = log_call_args[0]
+        assert expected_uuid in log_message
+        assert "GET" in log_message
+        assert expected_route in log_message
+
+    def test_Given_route_mocked_to_error_and_error_details_should_be_displayed__Then_uuid_in_log_and_response__and_details_in_response_and_log(
```

Ruff (0.12.2)

Function name `test_Given_route_mocked_to_error_and_error_details_should_be_displayed__Then_uuid_in_log_and_response__and_details_in_response_and_log` should be lowercase

(N802)